front/src/applications/operationalStudies/views/v2/SimulationResultsV2.tsx

```diff
@@ -1,23 +1,24 @@
 import React, { useEffect, useState, useRef, useMemo } from 'react';
 
 import { ChevronLeft, ChevronRight } from '@osrd-project/ui-icons';
+import { Manchette } from '@osrd-project/ui-manchette';
```

Comment:
It's a bit weird to import only Manchette, since the main component is not the Manchette but the SpaceTimeChart. It would have be nice to have the Manchette exported from the ui-space-time-chart module instead, since it won't be used without it, right ?
As a quick fix, you can use an alias like this:
```suggestion
import { Manchette as SpaceTimeChartWithManchette } from '@osrd-project/ui-manchette';
```
This way, when reading the code, we can know that the SpaceTimeChart is hidden in the Manchette

Reply:
done